swagger.yml

```
@@ -177,6 +177,266 @@ paths:
             $ref: '#/definitions/sniCertificate'
         default:
           $ref: '#/responses/error'
+  /accounts/{account_id}/env:
+    get:
+      tags:
+        - environmentVariables
+      operationId: getEnvVars
+      parameters:
+        - name: account_id
+          description: Scope response to account_id
+          type: string
+          required: true
+          in: path
+        - name: context_name
+          description: Filter by deploy context
+          type: string
+          enum:
+            - all
+            - dev
+            - branch-deploy
+            - deploy-preview
+            - production
+          required: false
+          in: query
+        - name: scope
+          description: Filter by scope
+          type: string
+          enum:
+            - builds
+            - functions
+            - runtime
+            - post_processing
+          required: false
+          in: query
+        - name: site_id
+          description: 'If specified, only return environment variables set on this site'
+          type: string
+          required: false
+          in: query
+      responses:
+        '200':
+          description: OK
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/envVar'
+        default:
+          $ref: '#/responses/error'
+      description: >-
+        [Beta] Returns all environment variables for an account or site. An
+        account corresponds to a team in the Netlify UI. To use this endpoint, 
+        opt into the beta environment variable experience using the Netlify UI.
```

[sand] 

Oh no, I'm sorry, I did it again...  I wrote `opt into` instead of `opt in to` in the comment I left for you. I'm going to leave suggested edits for you, hopefully that makes it easier for you to apply 🙏 

```suggestion
        opt in to the beta environment variable experience using the Netlify UI.
```